src/cmds/pbs_rsub.c

```diff
@@ -546,8 +546,8 @@ print_usage()
 	"                [-R start_time] [-E end_time] [-D duration] [-q destination]\n"
 	"                [-r rrule_expression] [-W otherattributes=value...] -l resource_list\n";
 
-	fprintf(stderr, usage);
-	fprintf(stderr, usag2);
+	fprintf(stderr, "%s", usage);
+	fprintf(stderr, "%s", usag2);
```

Comment:
Took me a second, but I can see why this is safer. We don't want the string we're trying to print to have any formatting directives. Thanks for fixing this.